core/merginapi.cpp

```diff
@@ -1244,8 +1258,7 @@ void MerginApi::saveAuthData()
   settings.setValue( "apiRoot", mApiRoot );
   settings.endGroup();
 
-  mUserAuth->saveAuthData();
-  mUserInfo->clear();
+  mUserAuth->saveData();
```

Comment:
This was probably not the best pace to clear the user info, indeed.
But are we now actually clearing the user info when the user auth changes?
If not, we could end up with mis-matching userInfo and userAuth, no?